plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -688,57 +694,56 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.OBSERVER, 0x676767);
         put(Blocks.OBSIDIAN, 0x0F0B19);
         put(Blocks.OCHRE_FROGLIGHT, 0xFCF8D5);
-        put(Blocks.ORANGE_BANNER, 0x8F7748);
+        put(Blocks.ORANGE_BANNER, 0x000000);
         put(Blocks.ORANGE_BED, 0xF17613);
+        put(Blocks.ORANGE_CANDLE_CAKE, 0x000000);
         put(Blocks.ORANGE_CANDLE, 0xDD660A);
-        put(Blocks.ORANGE_CANDLE_CAKE, 0xDD660A);
         put(Blocks.ORANGE_CARPET, 0xF17613);
-        put(Blocks.ORANGE_CONCRETE, 0xE06101);
         put(Blocks.ORANGE_CONCRETE_POWDER, 0xE38420);
+        put(Blocks.ORANGE_CONCRETE, 0xE06101);
         put(Blocks.ORANGE_GLAZED_TERRACOTTA, 0xA29258);
         put(Blocks.ORANGE_SHULKER_BOX, 0xEC6C0A);
+        put(Blocks.ORANGE_STAINED_GLASS_PANE, 0x000000);
         put(Blocks.ORANGE_STAINED_GLASS, 0xD87F33);
-        put(Blocks.ORANGE_STAINED_GLASS_PANE, 0xD07B30);
         put(Blocks.ORANGE_TERRACOTTA, 0xA25426);
         put(Blocks.ORANGE_TULIP, 0x5E8E1F);
-        put(Blocks.ORANGE_WALL_BANNER, 0x8F7748);
+        put(Blocks.ORANGE_WALL_BANNER, 0x000000);
         put(Blocks.ORANGE_WOOL, 0xF17613);
         put(Blocks.OXEYE_DAISY, 0xB7CC91);
         put(Blocks.OXIDIZED_COPPER, 0x53A486);
-        put(Blocks.OXIDIZED_CUT_COPPER, 0x509A7F);
         put(Blocks.OXIDIZED_CUT_COPPER_SLAB, 0x509A7F);
         put(Blocks.OXIDIZED_CUT_COPPER_STAIRS, 0x509A7F);
+        put(Blocks.OXIDIZED_CUT_COPPER, 0x509A7F);
         put(Blocks.PACKED_ICE, 0x8DB4FA);
         put(Blocks.PACKED_MUD, 0x8F6B50);
         put(Blocks.PEARLESCENT_FROGLIGHT, 0xF8F3F3);
         put(Blocks.PEONY, 0x817E8A);
         put(Blocks.PETRIFIED_OAK_SLAB, 0x6D5533);
-        put(Blocks.PINK_BANNER, 0x8F7748);
+        put(Blocks.PINK_BANNER, 0x000000);
         put(Blocks.PINK_BED, 0xEE8DAC);
+        put(Blocks.PINK_CANDLE_CAKE, 0x000000);
         put(Blocks.PINK_CANDLE, 0xD26890);
-        put(Blocks.PINK_CANDLE_CAKE, 0xD26890);
         put(Blocks.PINK_CARPET, 0xEE8DAC);
-        put(Blocks.PINK_CONCRETE, 0xD6658F);
         put(Blocks.PINK_CONCRETE_POWDER, 0xE59AB5);
+        put(Blocks.PINK_CONCRETE, 0xD6658F);
         put(Blocks.PINK_GLAZED_TERRACOTTA, 0xED9BB6);
         put(Blocks.PINK_SHULKER_BOX, 0xE87C9F);
+        put(Blocks.PINK_STAINED_GLASS_PANE, 0x000000);
         put(Blocks.PINK_STAINED_GLASS, 0xF27FA5);
-        put(Blocks.PINK_STAINED_GLASS_PANE, 0xE97BA0);
         put(Blocks.PINK_TERRACOTTA, 0xA24E4F);
         put(Blocks.PINK_TULIP, 0x649E50);
-        put(Blocks.PINK_WALL_BANNER, 0x8F7748);
+        put(Blocks.PINK_WALL_BANNER, 0x000000);
         put(Blocks.PINK_WOOL, 0xEE8DAC);
-        put(Blocks.PISTON, 0x5A5A5A);
         put(Blocks.PISTON_HEAD, 0x987E52);
-        put(Blocks.PLAYER_HEAD, 0x2A1C0C);
-        put(Blocks.PLAYER_WALL_HEAD, 0x2A1C0C);
+        put(Blocks.PISTON, 0x6E6961);
+        put(Blocks.PLAYER_HEAD, 0x000000);
```

Comment:
As mentioned on the other head, I do want these to show up on the map. But, since we don't know what texture one would have we should just color it brown like the default steve hair.